front-end/src/components/Characteristics/Characteristics.jsx

```diff
@@ -0,0 +1,51 @@
+import './Characteristics.css';
+import { useEffect, useState } from 'react';
+import { characteristicsData } from '../../mocks/data';
+import { Available } from '../Available';
+import { CHARACTERISTICS_TITLE } from '../../constants';
+
+const etDescItems = characteristicsData.characteristics;
+
+export const Characteristics = () => {
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    const handleScroll = () => {
+        const element = document.querySelector('.characteristics-section');
+        const position = element.getBoundingClientRect();
+
+        if (position.top < window.innerHeight - 100 && position.bottom >= 0) {
```

Comment:
should move 100 and 0 into global constant and naming it such as THIS_IS_NAME = 100